stripe/resource.py

```diff
@@ -114,6 +115,8 @@ def __init__(self, id=None, api_key=None, stripe_account=None, **params):
         if id:
             self['id'] = id
 
+        self.requestor = requestor
```

Comment:
You'd need to replace this with `object.__setattr__(self, 'requestor', requestor)`, or rename the variable to `_requestor`. Otherwise it's going to be treated as an API resource key (cf. https://github.com/ofek/stripe-python/blob/07ab2648a7f006893fd3337f5f939f0d3f89befc/stripe/resource.py#L127).